data/pcon_2024_v2.csv

```diff
@@ -0,0 +1,651 @@
+﻿PCON24CD,PCON24NM,PCON24NMW,ObjectId
```

Comment:
Is there any reason to keep the ObjectId field?

Reply:
Just to keep faithful to the original file from ONS, so we've not transformed it in anyway (and to be lazy...)